### ISSUE TITLE
boards (GitHub Projects) from key stakeholders to help prioritize issues

### ISSUE THREAD

**Issue by pdurbin** [astropy/astropy#9299]
### Description

@astrofrog hi! You have way more context than most for the following thread I started on the Dataverse mailing list:

-  boards for prioritizing issues (for each installation of Dataverse): https://groups.google.com/d/msg/dataverse-community/z8ugn2KzNSc/zt7COeRYBQAJ

I'd love to hear what you (and others!) think about this and if you think Astropy would benefit from this approach.

For the hacky implementation I threw together this weekend, please see https://github.com/pdurbin/druthers/issues/1

I'm happy to talk this out on Gitter or wherever.

Thanks!

**Comment by pllim** [comment]
cc @astropy/coordinators and @bsipocz

**Comment by astrofrog** [comment]
@pdurbin - thanks for pointing us to that discussion! I'm going to close this issue since we don't typically use issues for free-form discussions like this. I'll be in touch over Gitter and we can always decide to broaden the discussion to the astropy-dev mailing list.

**Comment by pdurbin** [comment]
@astrofrog you're welcome! The reason I reached out is that Astropy is pretty high up on the list at https://www.codetriage.com . Here's a screenshot from page two:

![Screen Shot 2019-09-30 at 1 48 13 PM](https://user-images.githubusercontent.com/21006/65902705-01b25580-e389-11e9-885e-6b98ce9855ac.png)

The project I work on (Dataverse) has a similar number of open issues (and perhaps similar pain) so I thought you might be interested in a new method I came up with to have the community help triage issues. I'm looking forward to chatting with you about it! It's certainly on topic for http://chat.dataverse.org so if you ever want to chat there that's fine too. Thanks!
